src/redux/selectors.js

```diff
@@ -55,11 +56,26 @@ const orders = {
   all: (state: StateType): OrdersType =>
     _.get(state, 'firestore.ordered.orders'),
 
+  allOrdered: (state: StateType): OrdersType =>
+    _.get(state, 'firestore.ordered.orders'),
+
   order: (state: StateType, { orderId }: { orderId: IdType }): OrderType =>
     _.get(state, `firestore.data.orders[${orderId}]`),
 
   items: (state: StateType, { orderId }: { orderId: IdType }): OrderItemsType =>
     _.get(state, `firestore.data.orders[${orderId}].items`),
+
+  fullItems: (
+    state: StateType,
+    { orderId }: { orderId: IdType }
+  ): OrderFullItemsType => {
+    const orderItems = orders.items(state, { orderId })
+    // $FlowFixMe
```

Comment:
Is it still relevant?

Reply:
Yep, dont have time to figure out how to fix this